apps/simple-trading-app/src/app/components/simple-market-list/simple-market-expires.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
praise (non-blocking): I like this solution for market expiry after all the conversations about it